src/main/resources/application.properties

```diff
@@ -72,3 +72,4 @@ google.singleDataProjectId=${GOOGLE_CLOUD_DATA_PROJECT}
 google.allowReuseExistingBuckets=false
 google.allowReuseExistingProjects=false
 management.health.probes.enabled=true
+management.endpoints.web.exposure.include=*
```

Comment:
What does the wildcard mean here?

Reply:
That exposes the management endpoints in spring boot.  They're not proxied out of the pod though